lib/Rx/Disposable/SerialDisposable.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Rx\Disposable;
+
+use Rx\DisposableInterface;
+
+class SerialDisposable implements DisposableInterface
```

Comment:
Tests added in https://github.com/davidwdan/Rx.PHP/commit/f3d391119489bbfa48d38bab661655f746b03c31